lib/delivery_sms.rb

```diff
@@ -0,0 +1,44 @@
+require 'twilio-ruby'
+
+class DeliverySMS
+
+  ONE_HOUR = 3600
+
+  def initialize
+    account_sid = ENV["TWILIO_ACCOUNT_SID"]
+    auth_token = ENV["TWILIO_AUTH_TOKEN"]
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+    @from = ENV["TWILIO_PHONE_NUMBER"]
+    @to = ENV["MY_PHONE_NUMBER"]
+  end
+
+  def send_confirmation
+
+    delivery_time = Time.new + ONE_HOUR
```

Comment:
Would be great to split the creation of the time string into a private method to simplify the send_confirmation method.